diff --git a/node_modules/react-native-wheely/lib/WheelPicker.js b/node_modules/react-native-wheely/lib/WheelPicker.js
index 575325b..8ca0fc2 100644
--- a/node_modules/react-native-wheely/lib/WheelPicker.js
+++ b/node_modules/react-native-wheely/lib/WheelPicker.js
@@ -32,7 +32,7 @@ const WheelPicker_styles_1 = __importDefault(require("./WheelPicker.styles"));
 const WheelPickerItem_1 = __importDefault(require("./WheelPickerItem"));
 const WheelPicker = ({ selectedIndex, options, onChange, selectedIndicatorStyle = {}, containerStyle = {}, itemStyle = {}, itemTextStyle = {}, itemHeight = 40, scaleFunction = (x) => Math.pow(1.0, x), rotationFunction = (x) => 1 - Math.pow(1 / 2, x), opacityFunction = (x) => Math.pow(1 / 3, x), visibleRest = 2, decelerationRate = 'fast', containerProps = {}, flatListProps = {}, }) => {
     const flatListRef = (0, react_1.useRef)(null);
-    const [scrollY] = (0, react_1.useState)(new react_native_1.Animated.Value(0));
+    const [scrollY] = (0, react_1.useState)(new react_native_1.Animated.Value((selectedIndex ?? 0) * itemHeight));
     const containerHeight = (1 + visibleRest * 2) * itemHeight;
     const paddedOptions = (0, react_1.useMemo)(() => {
         const array = [...options];
